public class ExceptionUser2 {

	public static void main(String[] args) {
		ExceptionTest test = new ExceptionTest();
//		test.call();
	}

}
